### PR TITLE
APC buffs

### PR DESCRIPTION
**Pull request by Kirshbia**
# About the pull request

What's changed
Module and base APC health increased dramatically. 
Boyars canon damage buffed and given more ammo
the frontal IFF gun has less spread and more ammo.

# Changelog
:cl:
qol: APC now feels a lot better to use.
balance: rebalanced the APC for PVE
/:cl:

### CODE REVIEW

**Comment by Kirshbia** [comment]
Slight adjustments made after showing off the changes on stream.

**Review by Kirshbia — COMMENTED** [review]
Some more slight adjustments. I always pictured people would go BRRRT on Boyars so made it a little easier to do and put the boyars at risk.

**Review by Kirshbia — COMMENTED** [review]
Just a little more after getting more feedback. I want to run a round using it after this change.

File: code/modules/vehicles/hardpoints/primary/dual_cannon.dm [hunk 1]
```diff
@@ -9,9 +9,9 @@
 	disp_icon_state = "dualcannon"
 	activation_sounds = list('sound/weapons/vehicles/dual_autocannon_fire.ogg')
 
-	damage_multiplier = 0.2
+	damage_multiplier = .15
```
Comment (FslashN) [inline code/modules/vehicles/hardpoints/primary/dual_cannon.dm:12]: Needs to be 0.15. Runtimes otherwise.

**Review by FslashN — DISMISSED** [review]
Small change to compile properly.

File: code/modules/vehicles/hardpoints/primary/dual_cannon.dm (hunk 1 above)
Comment (Kirshbia) [reply code/modules/vehicles/hardpoints/primary/dual_cannon.dm]: Got it in. :D I woke up today and wonder why I got emails it was failing. Fixed now.

**Review by morrowwolf — APPROVED** [review]

**Pull request merged**